cluster/local/kind.sh

```diff
@@ -72,7 +72,7 @@ case "${1:-}" in
   helm-upgrade)
     echo "copying image for helm"
     helm_tag="$(cat _output/version)"
-    copy_image_to_cluster ${BUILD_IMAGE} "${PROJECT_NAME}/${PROJECT_NAME}:${helm_tag}" "${KIND_NAME}"
+    copy_image_to_cluster ${BUILD_IMAGE} "xpkg.upbound.io/${PROJECT_NAME}/${PROJECT_NAME}:${helm_tag}" "${KIND_NAME}"
```

Comment:
Looks like you also brought in the change from the other pr

Reply:
Removed those from here.